client.go

```diff
@@ -725,6 +773,8 @@ func (c *Client) Do(req *Request) (*http.Response, error) {
 			if c.ResponseLogHook != nil {
 				// Call the response logger function if provided.
 				switch v := logger.(type) {
+				case ContextLogger:
+					c.ResponseLogHook(nil, resp)
```

Comment:
Why `nil` here?

Reply:
You are right, I will revise it, thank you